t/test-genome-DBs/homo_sapiens/funcgen/meta.txt

```diff
@@ -180,3 +180,6 @@
 752	\N	patch	patch_95_96_d.sql|add ReadFile to the enum of the ensembl_object_type
 753	\N	patch	patch_95_96_e.sql|Add description and release_version columns to regulatory_build table
 754	\N	patch	patch_95_96_f.sql|Modify binding_matrix_table
+755	\N	patch	patch_96_97_a.sql|schema_version
+756	\N	patch	patch_95_96_b.sql|Changed to text
+757	\N	patch	patch_95_96_c.sql|Added flag
```

Comment:
The patch order looks wrong here?!!

Reply:
I've reverted that commit and repatched the test dbs.
My funcgen checkout was not up to date, and looks like it was somehow fishy...
Thanks for noticing!